tests/conftest.py

```diff
@@ -1264,6 +1265,16 @@ def initialize_providers_manager():
     ProvidersManager().initialize_providers_configuration()
 
 
+@pytest.fixture(autouse=True)
+def create_swagger_ui_dir_if_missing():
+    """
+    The directory needs to exist to satisfy starlette attempting to register it as middleware
+    :return:
+    """
+    swagger_ui_dir = AIRFLOW_SOURCES_ROOT_DIR / "airflow" / "www" / "static" / "dist" / "swagger-ui"
+    swagger_ui_dir.mkdir(exist_ok=True, parents=True)
```

Comment:
I think better propagate it in the CI and raise an error if `"static" / "dist"` not available, it might prevent some serious error rather than silence it

Reply:
Good point.